src/components/UserNavbarItem/__tests__/item.test.tsx

```diff
@@ -74,4 +75,15 @@ describe('Given device type is desktop', () => {
     });
     expect(current_account).toBeInTheDocument();
   });
+
+  it('Should close the account dropdown when clicking outside of it', async () => {
+    const current_account_button = screen.getByRole('button', { name: /CR111111/i });
+    await userEvent.click(current_account_button);
```

Comment:
same question :) do we need await?

Reply:
Yes we do need await. We first want the click action to be finished before it searches for the item that would be visible after the click action. Test would otherwise fail.